src/code/templates/genome-playground.js

```diff
@@ -18,13 +18,15 @@ export default class GenomeContainer extends Component {
     let penView = <div className='columns bottom'>
                     <PenView orgs={ keptDrakes } width={500} columns={5} rows={1} tightenRows={20}/>
                   </div>;
-    
+
+    // we don't increment the score, so users always get top mark from playground
+    const kNoScoreIncrement = false;
```

Comment:
It's a constant intended to be passed as an argument to a function which accepts a boolean argument named `incrementMoves`. If the constant/variable were named `incrementMoves` as well, then the function call would look like:
```
onChromosomeAlleleChange(..., incrementMoves);
```
which could be read as a request to increment the moves rather than to _not_ increment the moves, as is actually the case. Without a constant, the code looks like:
```
onChromosomeAlleleChange(..., false);
```
which is uninterpretable without scrambling to look up the function definition. Passing a semantically meaningful constant to the argument named `incrementMoves` is intended to make the intent of the code more clear. Whether or not the precise name of the constant is as semantically meaningful as I originally could be a matter for further debate.